both/collections/registration.js

```diff
@@ -32,7 +32,7 @@ Registrations.attachSchema({
       });
 
       if(uniqueMatch && (this.isInsert || docId != uniqueMatch._id)) {
-        return "notUnique";
+        return "notUnique"; // TODO this has no message?!?
```

Comment:
You get 'Unknown validation error'.

What should the message be?

To editorialize: The validation message part of whatever package this is is really unimpressive.